pkg/ostree/index.html

```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<!--
+This file is part of Cockpit.
+
+Copyright (C) 2015 Red Hat, Inc.
+
+Cockpit is free software; you can redistribute it and/or modify it
+under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+Cockpit is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+-->
+<html>
+
+<head>
+  <title>Software Updates</title>
+  <meta charset="utf-8">
+  <link href="../base1/cockpit.css" rel="stylesheet">
+  <link href="../ostree/ostree.css" rel="stylesheet">
+  <script src="../base1/bundle.js"></script>
+  <script src="../ostree/ostree.js"></script>
+</head>
+<body>
```

Comment:
Body should be hidden until curtains != silent. Well either that, or just explictly show it without involving angularjs.